src/components/messages/message-item.js

```diff
@@ -47,12 +55,17 @@ const MessageItem = ({messageItem, me}) => {
               <div className="p-2 text-break overflow-auto bg-secondary bg-opacity-25">
                 {messageItem.message}
               </div>
+                <div>
```

Comment:
Let's not have the ability to received messages.

Reply:
Ok, can we discuss soft-deleting today? I don't understand how this can be implemented.

Reply:
Yes let's discuss it today over a call.